enhancements/machine-api/converting-machine-api-to-cluster-api.md

```diff
@@ -0,0 +1,891 @@
+---
+title: converting-machine-api-to-cluster-api
+authors:
+  - "@JoelSpeed"
+reviewers:
+  - "@damdo" # Cluster API and Machine API maintainer
+  - "@nrb" # Cluster API and Machine API maintainer
+  - "@mdbooth" # OpenStack Cluster API and Machine API maintainer
+  - "2uasimojo" # Hive maintainer
+approvers: 
+  - "@vincepri" # Cluster API maintainer within OpenShift
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-08-30
+last-updated: 2024-03-19
+tracking-link: 
+  - https://issues.redhat.com/browse/OCPCLOUD-1578
+see-also: []
+replaces: []
+superseded-by: []
+---
+...
```

Comment:
This is the YAML "document end" marker. It tells the parser to stop looking any further as part of the metadata lint.
Without this, I was getting parsers errors because of invalid mappings on line 36

CC @dhellmann you may want to be aware of this, the error doesn't manifest in `gopkg.in/yaml.v2` but does in `gopkg.in/yaml.v3`. Seems it was recently updated.